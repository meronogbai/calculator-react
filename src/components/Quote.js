const Quote = () => (
  <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Ratione, facilis?</p>
);

export default Quote;
