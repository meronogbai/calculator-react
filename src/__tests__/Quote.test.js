import { create } from 'react-test-renderer';
import Quote from '../components/Quote';

test('renders correctly', () => {
  const tree = create(<Quote />).toJSON();
  expect(tree).toMatchSnapshot();
});
