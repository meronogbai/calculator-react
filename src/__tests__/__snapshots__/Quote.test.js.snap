// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<p>
  Lorem ipsum dolor sit amet consectetur adipisicing elit. Ratione, facilis?
</p>
`;
